uaclient/tests/test_ua_update_messaging.py

```diff
@@ -192,38 +302,62 @@ def test_apt_templates_written_for_disabled_services(
             no_pkgs_file.strpath,
             motd_pkgs_file.strpath,
             motd_no_pkgs_file.strpath,
-            no_warranty_file.strpath,
         )
         if expect_messages:
             assert (
                 MESSAGE_DISABLED_APT_PKGS_TMPL.format(
-                    title="UA Apps: ESM",
-                    pkg_num="{ESM_APPS_PKG_COUNT}",
-                    pkg_names="{ESM_APPS_PACKAGES}",
-                    url=BASE_ESM_URL,
+                    title=title,
+                    pkg_num=pkg_count_var,
+                    pkg_names=pkg_names_var,
+                    eol_release=eol_release,
+                    url=url,
                 )
                 == pkgs_file.read()
             )
             assert (
-                MESSAGE_DISABLED_MOTD_NO_PKGS_TMPL.format(
-                    title="UA Apps: ESM", url=BASE_ESM_URL
-                )
+                MESSAGE_DISABLED_MOTD_NO_PKGS_TMPL.format(title=title, url=url)
                 == no_pkgs_file.read()
             )
         else:
             assert False is os.path.exists(pkgs_file.strpath)
             assert False is os.path.exists(no_pkgs_file.strpath)
 
     @pytest.mark.parametrize(
-        "contract_status, remaining_days, is_active_esm",
+        "contract_status, remaining_days, is_active_esm, platform_info",
         (
-            (ContractExpiryStatus.ACTIVE, 21, True),
-            (ContractExpiryStatus.ACTIVE_EXPIRED_SOON, 10, True),
-            (ContractExpiryStatus.EXPIRED_GRACE_PERIOD, -5, True),
-            (ContractExpiryStatus.EXPIRED, -20, True),
-            (ContractExpiryStatus.EXPIRED, -20, False),
+            (
+                ContractExpiryStatus.ACTIVE,
+                21,
+                True,
+                {"series": "xenial", "releaes": "16.04"},
```

Comment:
Minor typo here and where it was copied below
```suggestion
                {"series": "xenial", "release": "16.04"},
```